lib/rubocop/cop/rails/redundant_receiver_in_with_options.rb

```diff
@@ -42,14 +42,14 @@ class RedundantReceiverInWithOptions < Cop
             ...)
         PATTERN
 
-        def_node_search :assoc_has_redundant_receiver, <<-PATTERN
+        def_node_search :rails_assoc_with_redundant_receiver, <<-PATTERN
```

Comment:
Since this is a Rails cop, does it need to be that particular that it's a Rails association?